bin/plugin/commands/performance.js

```diff
@@ -205,7 +207,7 @@ async function runPerformanceTests( branches ) {
 	);
 
 	log( '>> Starting the WordPress environment' );
-	await runShellScript( 'npm run wp-env start' );
+	await runShellScript( 'npm run wp-env start', environmentDirectory );
```

Comment:
This is the main issue we had, the environment was not updating when switching branches on the temporary clone